undefined_syms.txt

```diff
@@ -564,3 +564,9 @@ D_04032C90 = 0x04032C90;
 // z_oceff_storm
 D_04032490 = 0x04032490;
 D_04032890 = 0x04032890;
+
+// code_800AD920
+D_0E000000 = 0x0E000000;
+
+// z_vismono
+D_0F000000 = 0x0F000000;
```

Comment:
These shouldn't actually be symbols. Or at least I don't see where they would be defined if they were.